docs/SPECIFICATION.md

```diff
@@ -277,19 +280,27 @@ message CastId {
   uint64 fid = 1;                           // Fid of the cast's author
   bytes hash = 2;                           // Message.hash value of the cast
 }
+
+message Embed {
+  oneof embed {
+    string url = 1;
+    CastId cast_id = 2;
+  }
+}
 ```
 
 A CastAddBody in a message `m` is valid only if it passes these validations:
 
 1. `m.signature_scheme` must be `SIGNATURE_SCHEME_ED25519`.
 2. `m.data.type` must be `MESSAGE_TYPE_CAST_ADD`.
 3. `m.data.body.type` must be `CastAddBody`.
-4. `m.data.body.embeds` can contain up to 2 valid utf8 strings are between 1 and 256 bytes inclusive.
+4. `m.data.body.embeds_deprecated` can contain up to 2 valid utf8 strings are between 1 and 256 bytes inclusive.
```

Comment:
Add a note here that casts can only have `embeds_deprecated` if the message timestamp is before farcaster timestamp 73612800 (5/3/23 00:00 UTC)